pipeline-runner/R/qc-6-integrate_scdata.R

```diff
@@ -163,6 +166,17 @@ run_seuratv4 <- function(scdata, config) {
   for (i in 1:length(data.split)) {
     data.split[[i]] <- Seurat::NormalizeData(data.split[[i]], normalization.method = normalization, verbose = FALSE)
     data.split[[i]] <- Seurat::FindVariableFeatures(data.split[[i]], nfeatures = nfeatures, verbose = FALSE)
+    # PCA needs to be run also here
+    # otherwise when running FindIntegrationAnchors() with reduction="rpca" it will fail because no "pca" is present
+    # QUESTION: should we run it also after the integration (as the code currently does) or not?
```

Comment:
this does not go here. part of the ticket is to research and talk to people about things like these, and make a decision!

Reply:
> I think I would leave it as is. The PCA here is run on each sample separately. While after integration it is run on the complete and integrated dataset (AFAIK you can't combine PCAs done on different batches of observations). And those are the PCs then used for the UMAP calculation. @alexvpickering could probably confirm if this makes sense.

I agree, that comment was to be extra cautious that I was not breaking anything. I deleted it and I will resolve this conversation. But if Alex has a different opinion I will be happy to discuss this with him